flask_common/documents.py

```diff
@@ -75,6 +75,12 @@ def save(self, *args, **kwargs):
             self.date_updated = now
         return super(DocumentBase, self).save(*args, **kwargs)
 
+    def modify(self, *args, **kwargs):
```

Comment:
If the document doesn't have a pk, we throw `InvalidDocumentError` in `Document.modify()`, so I don't think this is ever a valid scenario.